### PR TITLE
Automated Workflow Executions: Update

### PR DESCRIPTION
**Pull request by joshsadam**
## What does this PR do and why?
_Describe in detail what your merge request does and why._

Allow an owner to update the parameters used for running a pipeline

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

Edit Button:
image

**Light**
image

**Dark**
image

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Created an automated workflow execution within a project
2. Within the table, click on the edit button on the far right for the automated workflow execution that you just created.
3. Updated whichever parameters you want and click `Update` button.
4. The changes should be reflected in the table, or click `Edit` again and the changes you made should be there.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [X] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

File: app/views/projects/automated_workflow_executions/_edit_dialog.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<%= viral_dialog(open: true, size: :extra_large) do |dialog| %>>
+  <%= dialog.with_header(title: t(".title", workflow_name: @automated_workflow_execution.name)) %>
```
Comment (ericenns) [inline app/views/projects/automated_workflow_executions/_edit_dialog.html.erb:2]: **nitpick (blocking):** If `name` isn't set can you pass `id` instead.

File: config/locales/en.yml [hunk 2]
```diff
@@ -863,6 +864,9 @@ en:
         title: Workflow Selection
       new_automated_workflow_execution_modal:
         title: "%{workflow} parameters"
+      edit_dialog:
+        title: "Editing Automated Workflow Execution: %{workflow_name}"
+        submit: Save changes
```
Comment (ericenns) [inline config/locales/en.yml:869]: **nitpick (blocking):** I believe `submit` is unused here, if so can you remove it.

**Review by ericenns — CHANGES_REQUESTED** [review]
Looks good, just a few minor comments

File: app/views/projects/automated_workflow_executions/_edit_dialog.html.erb (hunk 1 above)
Comment (joshsadam) [reply app/views/projects/automated_workflow_executions/_edit_dialog.html.erb]: Updated in d4fb7f7

File: config/locales/en.yml (hunk 2 above)
Comment (joshsadam) [reply config/locales/en.yml]: Removed in 2fa5e21

**Review by ericenns — APPROVED** [review]
Looks good to me!

**Pull request merged**